New package: OrdinaryDiffEqCore v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: OrdinaryDiffEqCore
- Repository: https://github.com/SciML/OrdinaryDiffEq.jl
- Created by: @ChrisRackauckas
- Version: v1.0.0
- Commit: 65946a86b8acd900ec38dd5e80f3c51d901f7771
- Reviewed by: @ChrisRackauckas
- Reference: https://github.com/SciML/OrdinaryDiffEq.jl/issues/845#issuecomment-2284972157
- Description: High performance ordinary differential equation (ODE) and differential-algebraic equation (DAE) solvers, including neural ordinary differential equations (neural ODEs) and scientific machine learning (SciML)
<!-- 45a4cd22be5b6a15fc3c6d377732d7af7690b65209effd7967e2547cc3da18458c7923a2b4aa63c6d000bd0b2a57d75eaa5db7e7ea7e1a8c9956099eb2c0a3f6e2e615055325a3da9238aa5aaee7fda2feb618b13a749af5a3aec9466a153ad7f9ff0a06bf39123aa6ac0bda32b5cfbccf04b0a94ab598b5392c9db607531d478ea75f8455b04ad2fdf43b48f4cdfa8225c023b6cfb946aea50b1a0ef78e913fe07e9aaab5b6d877226812b9916965c2db5f225d23e39a7e653630076012577a -->

ViralBShah commented:
@ChrisRackauckas Perhaps a license has to be added to all the packages in the repo?

ChrisRackauckas commented:
oh that's annoying. Yeah I can add it to each.

ViralBShah commented:
I don't know if there is a way to have these packages look one level further up when in a multi-package repo. Perhaps this is in @KristofferC's wheelhouse?

In any case, it may be quicker to add the licenses.

ChrisRackauckas commented:
I'll just add the licenses but report that this could probably be improved. I generally need to write an issue about how this should all be done using Julia submodules but there are missing features that makes this use packages instead. I haven't gotten around to writing all of this though. But for simplicity, I'll just copy paste the same license file 20 times and call it a day for now.

JuliaRegistrator committed:
New package: OrdinaryDiffEqCore v1.0.0

UUID: bbf590c4-e513-4bbe-9b18-05decba2e5d8
Repo: https://github.com/SciML/OrdinaryDiffEq.jl.git
Tree: 4583d65870823ca6a323fc3f44d92dabe5133fbb

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

fredrikekre commented:
Since only the subfolder is actually shipped to users there has to be a license there.

ChrisRackauckas commented:
I see, makes sense.
